components/elm/src/main/elm_initializeMod.F90

```diff
@@ -9,12 +9,12 @@ module clm_initializeMod
   use shr_log_mod      , only : errMsg => shr_log_errMsg
```

Comment:
Comments containing clm also need to be cleaned up.

Reply:
Opened a new issue for it.